config/routes.rb

```diff
@@ -447,6 +447,7 @@
 
   get "stylesheets/:name.css.map" => "stylesheets#show_source_map", constraints: { name: /[-a-z0-9_]+/ }
   get "stylesheets/:name.css" => "stylesheets#show", constraints: { name: /[-a-z0-9_]+/ }
+  get "javascripts/:digest.js" => "javascripts#show", constraints: { name: /[-a-z0-9_]+/ }
```

Comment:
I'm surprised you don't have to escape values in the bracketed area. Usually I would do `\-` and `\_` but this seems to work?

Reply:
Ah thanks, will double check by adding a test, and also test for the security aspect.